src/experimental/barrelroll.cpp

```diff
@@ -2,32 +2,34 @@
 #include <ros/ros.h>
 #include <cmath>
 #include <string>
-#include "helpers/AngularVelocity.h"
+
```

Comment:
I thought this file had comments. Add comments. Especially the while loop in main.